assets/js/index.js

```diff
@@ -154,6 +155,16 @@ $(document).ready(function () {
     }
   });
 
+  $("body").click(function(event) {
+    let sidebar = $(".sidebar") ;
+    if( $(window).width() <= 640 && event.target.className.split(" ",1)!="sidebar" && event.target.className.split(" ",1)!="fas" && sidebar.hasClass("animate__slideInLeft") && (!sidebar.hasClass("hide-sidebar"))){
```

Comment:
Why is it 640 instead of 767?

Reply:
because you specified here
``` $(".scroll-top").click(function () 
    let scrollSpeed = 100; /* Default */
    if ($(window).width() <= 640)
      scrollSpeed = 210; /* For small (mobile) Screens */
```

Reply:
No this is for a different issue. Please change it to 767, as it is the size of the screen where sidebar appears by clicking on the hamburger menu.